src/Parser.zig

```diff
@@ -1796,12 +1804,10 @@ fn recordDecls(p: *Parser) Error!void {
 fn recordDeclarator(p: *Parser) Error!bool {
     const attr_buf_top = p.attr_buf.len;
     defer p.attr_buf.len = attr_buf_top;
+    try p.attributeSpecifier(); // .record
     const base_ty = (try p.specQual()) orelse return false;
 
     while (true) {
-        const this_decl_top = p.attr_buf.len;
-        defer p.attr_buf.len = this_decl_top;
-
```

Comment:
These are here on purpose, multiple declarators on the same declaration specifier do not share attributes:
```c
struct Foo {
    int a __attribute__((packed)), b; // only 'a' is packed
};
```